connect/connect-c8/element-template-c8/src/main/java/io/miragon/miranum/connect/adapter/in/c8/elementtemplates/schema/Binding.java

```diff
@@ -0,0 +1,291 @@
+    {
+        return type;
+    }
+
+    /**
+     * binding type
+     * <p>
+     * The type of a property binding
+     * (Required)
+     */
+    @JsonProperty("type")
+    public void setType(Binding.Type type)
+    {
+        this.type = type;
+    }
+
+    public Binding withType(Binding.Type type)
+    {
+        this.type = type;
+        return this;
+    }
+
+    /**
+     * binding name
+     * <p>
+     * The name of a property binding
+     */
+    @JsonProperty("name")
+    public String getName()
+    {
+        return name;
+    }
+
+    /**
+     * binding name
+     * <p>
+     * The name of a property binding
+     */
+    @JsonProperty("name")
+    public void setName(String name)
+    {
+        this.name = name;
+    }
+
+    public Binding withName(String name)
+    {
+        this.name = name;
+        return this;
+    }
+
+    /**
+     * binding source
+     * <p>
+     * The source value of a property binding (zeebe:output)
+     */
+    @JsonProperty("source")
+    public String getSource()
+    {
+        return source;
+    }
+
+    /**
+     * binding source
+     * <p>
+     * The source value of a property binding (zeebe:output)
+     */
+    @JsonProperty("source")
+    public void setSource(String source)
+    {
+        this.source = source;
+    }
+
+    public Binding withSource(String source)
+    {
+        this.source = source;
+        return this;
+    }
+
+    /**
+     * binding key
+     * <p>
+     * The key value of a property binding (zeebe:taskHeader)
+     */
+    @JsonProperty("key")
+    public String getKey()
+    {
+        return key;
+    }
+
+    /**
+     * binding key
+     * <p>
+     * The key value of a property binding (zeebe:taskHeader)
+     */
+    @JsonProperty("key")
+    public void setKey(String key)
+    {
+        this.key = key;
+    }
+
+    public Binding withKey(String key)
+    {
+        this.key = key;
+        return this;
+    }
+
+    @Override
+    public String toString()
+    {
+        StringBuilder sb = new StringBuilder();
+        sb.append(Binding.class.getName()).append('@').append(Integer.toHexString(System.identityHashCode(this))).append('[');
+        sb.append("type");
+        sb.append('=');
+        sb.append(((this.type == null) ? "<null>" : this.type));
+        sb.append(',');
+        sb.append("name");
+        sb.append('=');
+        sb.append(((this.name == null) ? "<null>" : this.name));
+        sb.append(',');
+        sb.append("source");
+        sb.append('=');
+        sb.append(((this.source == null) ? "<null>" : this.source));
+        sb.append(',');
+        sb.append("key");
+        sb.append('=');
+        sb.append(((this.key == null) ? "<null>" : this.key));
+        sb.append(',');
+        if (sb.charAt((sb.length() - 1)) == ',')
+        {
+            sb.setCharAt((sb.length() - 1), ']');
+        } else
+        {
+            sb.append(']');
+        }
+        return sb.toString();
+    }
+
```

Comment:
And do we need hashcode and equals? we can also just use lombok to add getter and setter

Reply:
Using lombok is a good idea.